### ISSUE TITLE
[Suggestion]: Add Botany Pots Tiers to OceanBlock (and other packs, please)

### ISSUE THREAD

**Issue by Lordmau5** [FTBTeam/FTB-Modpack-Issues#150]
### Modpack

FTB OceanBlock

### Is your suggestion related to a problem? Please describe.

So I've gotten to endgame and it requires LOTS of resources (or a very patient playstyle) to get all the singularities.

I've managed to setup Compact Machines with lots of Botany Pots and their corresponding crafting requirements but the more of those are required the more lag is being caused...

### Mod suggestion

I stumbled upon this mod: https://www.curseforge.com/minecraft/mc-mods/botany-pots-tiers

Basically it adds 2 new tiers (3 if the creative-only Enchanted Golden Apple would be craftable) that have both higher speed AND higher output per iteration.

With this I don't need to have one Compact Machine filled to the brim with Inferium Essence anymore but rather only a few rows and they're already able to surpass the processing speed of the normal Botany Pots (with the 6x speed / 6x output versions)

**Comment by Sunekaer** [comment]
Thanks for the suggestion, this wont be added.
Tho there will be a powered version of the Pots in the next update that will generate resources faster
